website/src/content/docs/pt-br/formatter/option-philosophy.md

```diff
@@ -1,38 +1,38 @@
+O Biome se orgulha de ter alcançado uma compatibilidade tão alta com o Prettier, e de fazer com que a migração seja o mais simples possível, mas isso também vem com ressalvas semelhantes.
 
-## Existing Options
+## Opções Existentes
 
-Biome started out with a strict subset of configuration options, targeting the most common and contentious style guidelines in the JavaScript ecosystem: indent styles (tabs vs spaces), indent widths (2 spaces to equal a tab, or 4?), and enforced semicolons. Adding options for these points was considered sufficient enough to address most people’s needs, and there was no strong consideration for adding any others.
+O Biome começou com um subconjunto restrito de opções de configuração, visando às diretrizes de estilo mais comuns e controversas no ecossistema JavaScript: tipo de identação (tabulações vs. espaços), larguras da identação (2 espaços para igualar uma tabulação ou 4?) e pontos-e-vírgulas obrigatórios.  Adicionar opções para esses pontos foi considerado suficiente para atender às necessidades da maioria das pessoas, e não houve nenhuma consideração forte para adicionar mais outros.
 
-Leaning on the [Prettier Option Philosophy](https://prettier.io/docs/en/option-philosophy), Biome had the chance to start fresh and avoid the pitfalls that Prettier had fallen into with its other existing options, like `--bracket-same-line` and `--arrow-parens`:
+Tomando por base [a filosofia de opções do Prettier](https://prettier.io/docs/en/option-philosophy), Biome tinha a oportunidade de começar do zero e evitar as armadilhas nas quais o Prettier havia caído com algumas de suas opções existentes, como `--bracket-same-line` e `--arrow-parens`:
 
-> …[these] are not the type of options we’re happy to have. They cause a lot of bike-shedding in teams, and we’re sorry for that. Difficult to remove now, these options exist as a historical artifact and should not motivate adding more options (“If *those* options exist, why can’t this one?”).
+> …[essas] não são opções que gostamos de oferecer. Elas causam muitas discussões inúteis nas equipes, e lamentamos por isso. Difíceis de remover agora, essas opções existem como um artefato histórico e não devem motivar a adição de mais opções ("Se *essas* opções existem, por que esta não pode existir também?").
 
-However, when the [Prettier Challenge was announced](https://twitter.com/Vjeux/status/1722733472522142022), Biome decided to accept the challenge, which required implementing all of the configuration options that Prettier had to achieve full compatibility.
+No entanto, quando o [Prettier Challenge foi anunciado](https://twitter.com/Vjeux/status/1722733472522142022), o Biome decidiu aceitar o desafio, o que exigia a implementação de todas as opções de configuração que o Prettier já oferecia para obter compatibilidade total.
```

Comment:
```suggestion
No entanto, quando o [Desafio do Prettier foi anunciado](https://twitter.com/Vjeux/status/1722733472522142022), o Biome decidiu aceitar o desafio, o que exigia a implementação de todas as opções de configuração que o Prettier já oferecia para obter compatibilidade total.
```

Reply:
It doesn't make sense to translate the name of the competition.

Reply:
We translated it in home page, so lets translate here too.

Reply:
For what it's worth, I still don't think it makes sense to translate the name of the competition, but I won't insist on it.